python_modules/dagster/dagster_tests/core_tests/test_stdout.py

```diff
@@ -33,10 +33,16 @@ def test_stdout():
     step_key = compute_steps[0]
     logs = instance.compute_log_manager.read_logs(result.run_id, step_key)
     assert logs.stdout.data == HELLO_WORLD + SEPARATOR
-    assert 'dagster - DEBUG - \n       dagster_event = {"event_specific_data":' in logs.stderr.data
+    assert (
+        'dagster - DEBUG - \n       dagster_event = {"event_specific_data":'
+        in logs.stderr.data.replace('\r\n', '\n')
```

Comment:
Wow, themoreyouknow.

Potentially could be useful to extract into a test utility function, along with a comment linking: 
https://softwareengineering.stackexchange.com/questions/29075/difference-between-n-and-r-n